admin/data/sql/migrations/110.do.create-question-reader-reasons.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE [mtc_admin].[questionReaderReasons] (
+  id int IDENTITY (1,1) NOT NULL,
+  createdAt datetimeoffset(3) NOT NULL DEFAULT GETUTCDATE(),
+  updatedAt datetimeoffset(3) NOT NULL DEFAULT GETUTCDATE(),
+  version rowversion,
+  displayOrder smallint NOT NULL,
+  description nvarchar(50) NOT NULL,
+  code char(3) NOT NULL,
+  CONSTRAINT [PK_questionReaderReasons] PRIMARY KEY CLUSTERED ([id] ASC)
+    WITH (
+      PAD_INDEX = OFF,
+      STATISTICS_NORECOMPUTE = OFF,
+      IGNORE_DUP_KEY = OFF,
+      ALLOW_ROW_LOCKS = ON,
+      ALLOW_PAGE_LOCKS = ON
+    ),
+  CONSTRAINT [questionReaderReasons_code_uindex] UNIQUE([code])
+)
```

Comment:
Should we have a convention that require's semi-colons to terminate SQL statements?  Not having one looks odd to me, and I wonder if it could cause trouble.

Reply:
commas are correct here, as its part of the create table statement.
very different to how i would usually create constraints, which would be separate statements.

maybe Georges SQL tool does it this way